06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: HTML, CSS & JS
+- **Deliver**: a folder containing the template for the proposed frontend of your project
+- **With**: *independently*
+
+#### Output
+[HTML and CSS for the app](https://github.com/andela-aawa/inverted-index/pull/18)
```

Comment:
Great Job!

The search however seems out of scope with the rest of the functionalities (i.e. Create Index, Upload).

It's also far away from the displayed result. It'll be too difficult to assume those functionalities go together.